unit_testing/myapp/test_models.py

```diff
@@ -1,15 +1,27 @@
+from django.core.cache import cache
 from django.test import TestCase
 from myapp.models import Person
 
 
 class PersonModelTest(TestCase):
-    def setUp(self):
-        Person.objects.create(first_name="Mehroze", last_name="Zahid")
-        Person.objects.create(first_name="Hello", last_name="World")
+    # def setUp(self):
+    #     Person.objects.create(first_name="Mehroze", last_name="Zahid")
+    #     Person.objects.create(first_name="Hello", last_name="World")
+    #
+    # def test_person_name(self):
+    #     person_1 = Person.objects.get(first_name="Mehroze")
+    #     person_2 = Person.objects.get(first_name="Hello")
+    #
+    #     self.assertEqual(person_1.last_name, "Zahid")
+    #     self.assertEqual(person_2.last_name, "World")
 
-    def test_person_name(self):
-        person_1 = Person.objects.get(first_name="Mehroze")
-        person_2 = Person.objects.get(first_name="Hello")
+    def test_cache(self):
+        self.fname = 'Mehroze'
```

Comment:
Have removed self from variables, however must self be removed from the function entirely? If so, how should the self.assertNumQueries() function be called?